src/Tools/dotnet-monitor/StartupHook/StartupHookValidator.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Microsoft.Diagnostics.Monitoring.WebApi;
+using Microsoft.Diagnostics.NETCore.Client;
+using Microsoft.Diagnostics.Tools.Monitor.LibrarySharing;
+using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Diagnostics.Tools.Monitor.StartupHook
+{
+    internal sealed class StartupHookValidator
+    {
+        private const string StartupHookFileName = "Microsoft.Diagnostics.Monitoring.StartupHook.dll";
+        private const string StartupHookTargetFramework = "net6.0";
```

Comment:
So will we always be using v6 for the startup hook, since that reflects the minimum target process this feature would support?

Reply:
That is the intent; I don't want to have to ship multiple TFMs for the startup hook unless we really need it for some technical reason.